src/package/abi/Microsoft.Windows.AbiWinRT.props

```diff
@@ -0,0 +1,58 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+        <!-- Only do this for MSBuild versions below 16.0
+             as it is since done automatically, see https://github.com/microsoft/msbuild/pull/3605-->
+        <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)'  &lt;= '15'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+        
+        <UsingAbiWinRT>true</UsingAbiWinRT>
+    </PropertyGroup>
+
+     <PropertyGroup>
+        <PreferredToolArchitecture>x64</PreferredToolArchitecture>
+        <CanReferenceWinRT>true</CanReferenceWinRT>
+        <IsNativeLanguage>true</IsNativeLanguage>
+        <!-- This causes VS to add the platform WinMD references for the target SDK -->
+        <WinMDAssembly>true</WinMDAssembly>
+        <!--Set a value to prevent SDK's uap.props from setting it and pointing to the wrong headers-->
+        <AbiWinRT_IncludePath>PreventSdkUapPropsAssignment</AbiWinRT_IncludePath>
+    </PropertyGroup>
+
+    <ItemDefinitionGroup>
+        <ClCompile>
+            <CompileAsWinRT Condition="'%(ClCompile.CompileAsWinRT)' == ''">false</CompileAsWinRT>
+        </ClCompile>
+        <Midl Condition="'$(AbiWinRTModernIDL)' != 'false'">
```

Comment:
can you remove AbiWinRTModernIDL and only add it if necessary later (cppwinrt needed it) - just assume modern is all that's supported?

Reply:
the scenarios for the abi nuget are likely to be much narrower than for cppwinrt, so the need for this degree of flexibility is much lower.

Reply:
now that I think of it, if this nuget is for consumption only, then all midl/mdmerge configuration should go away (like clcompile/link) - you should only be scanning for referenced winmds to generated headers

Reply:
going to just resolve all these midl related ones since we agreed a different nuget package is appropriate for that